src/queue/bookmark/index.ts

```diff
@@ -0,0 +1,20 @@
+import { NativeConnection, Worker } from '@temporalio/worker';
+import { createActivities } from './activities';
+import createOrGetConnection from '../../db';
+import { WorkflowQueue } from '../common';
+import { logger } from '../../logger';
+
+export async function run() {
+  const connection = await NativeConnection.connect({
+    address: 'host.docker.internal:7233',
+  });
+  const dbCon = await createOrGetConnection();
+  const worker = await Worker.create({
+    connection,
+    workflowsPath: require.resolve('./workflows'),
```

Comment:
You can't do this with normal path eg. `__dirname` or `path.resolve`? `require` will bug us if we ever move to ES modules.

Reply:
I can try, let's see how it goes.

Reply:
Did not work 😢 

We could probably use `import.meta` but it requires `es2020`.